pkg/csi/cinder/controllerserver.go

```diff
@@ -103,6 +103,14 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 
 	// Volume Create
 	properties := map[string]string{"cinder.csi.openstack.org/cluster": cs.Driver.cluster}
+	//Tag volume with metadata if present: https://github.com/kubernetes-csi/external-provisioner/pull/399
+	for k, v := range req.Parameters {
+		for _, mKey := range []string{"csi.storage.k8s.io/pvc/name", "csi.storage.k8s.io/pvc/namespace", "csi.storage.k8s.io/pv/name"} {
+			if k == mKey {
+				properties[k] = v
+			}
+		}
+	}
```

Comment:
```suggestion
	for _, mKey := range []string{"csi.storage.k8s.io/pvc/name", "csi.storage.k8s.io/pvc/namespace", "csi.storage.k8s.io/pv/name"} {
		if v, ok := req.GetParameters()[mKey]; ok {
			properties[mKey] = v
		}
	}
```

Reply:
lgtm, but I would simplify this code a little.

Reply:
Thanks, I simplified as suggested.